tests/specs/add.js

```diff
@@ -132,3 +133,57 @@ test('adds multiple objects to db', function (t) {
     t.is(res.total_rows, 3, 'puts docs')
   })
 })
+
+test('store.add(object) makes createdAt and updatedAt timestamps', function (t) {
+  t.plan(4)
+
+  var clock = sinon.useFakeTimers('Date')
```

Comment:
Do you know how this works internally? Maybe we don't need to pull in the entire `sinon` package, if faking date is the only thing we need?